### DISCUSSION TITLE
What are the parameters pyannote/speaker-diarization is running with ?

### DISCUSSION THREAD

**Discussion by amitli1** [pyannote/pyannote-audio#1292] [Development]
When creating` speaker-diarization` pipeline:
`Pipeline.from_pretrained("pyannote/speaker-diarization")`
 How can I get the parameters of each the building blocks running with ?

**Comment by hbredin** [comment]
See https://huggingface.co/pyannote/speaker-diarization/blob/main/config.yaml and/or run `pipeline.parameters(instantiated=True)`.
